kani-driver/src/list/output.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// This module handles outputting the result for the list subcommand
+
+use std::{
+    cmp::max,
+    collections::{BTreeMap, BTreeSet},
+    fs::File,
+    io::BufWriter,
+};
+
+use crate::{list::Totals, version::KANI_VERSION};
+use anyhow::Result;
+use colour::print_ln_bold;
+use kani_metadata::ContractedFunction;
+use serde_json::json;
+
+// Represents the version of our JSON file format.
+// Increment this version (according to semantic versioning rules) whenever the JSON output format changes.
+const FILE_VERSION: &str = "0.1";
+const JSON_FILENAME: &str = "kani-list.json";
+
+/// Construct the table of contracts information.
+fn construct_contracts_table(
```

Comment:
Earlier commits for this PR actually used a crate like that. I decided to change it to this because I thought it would be really nice to have it closer to Markdown. That way, when we're sharing the table format to track our verify std goal, we can format Markdown tables in documents instead of having screenshots of huge tables.
Since the code for this earlier version is committed, it would be really easy for me to change it back. I think for now we can see how much we actually take advantage of it being Markdown-adjacent and go from there?